### DISCUSSION TITLE
got 408 error when calling getModuleEventsByEventType

### DISCUSSION THREAD

**Discussion by particle4dev** [aptos-labs/aptos-developer-discussions#472] [Questions]
### Discord user ID

namhoang28

### Describe your question in detail.

I encountered the error **408 Request Timed Out: Upstream took longer than 10000ms to respond** when calling `getModuleEventsByEventType` from **@aptos-labs/ts-sdk**.

Here's a sample code:

```typescript
const events = await client.getModuleEventsByEventType({
  eventType: '0x1::coin::DepositEvent',
  options: {
    limit: 10,
    offset: 0, // get the latest events
    orderBy: [
      {
        transaction_version: 'desc' as OrderByValue,
      },
    ],
  },
});
```

Has anyone else encountered a similar issue?

It was working fine a few days ago, and the API doesn't seem to have any issues when I call it directly from **GraphQL Editor**. By the way, I'm using the **testnet** environment.

### Which operating system are you using?

N/A

### Which SDK or tool are you using? (if any)

TypeScript SDK

**Comment by BounebRayan** [comment]
Hello, did you find a solution ?
